awx/README.md

```diff
@@ -1,9 +1,20 @@
+<!--

```

Comment:
I'm using [`helm-docs`](https://github.com/norwoodj/helm-docs) to generate this file based on `REAMDE.md.gotmpl`